### ISSUE TITLE
Colorbox close button focus ring on is not visible in Firefox (win & osX)

### ISSUE THREAD

**Issue by regan-sarwas** [regan-sarwas/map-journal-storytelling-template-js#11]
The close button/control gets focus and responds to the enter key, but the focus ring is not visible.
